packages/common/src/search/_internal/discussionsSearchChannelsHelpers/index.ts

```diff
@@ -0,0 +1,86 @@
+import { IHubSearchOptions, IHubSearchResponse, IQuery } from "../../types";
+import HubError from "../../../HubError";
+import {
+  IChannel,
+  IPagedResponse,
+  ISearchChannels,
+  ISearchChannelsParams,
+} from "../../../discussions";
+import { discussionsSearchChannels } from "..";
+
```

Comment:
I think you could / should put all these in the `discussionsSearchChannels.ts` file - does not seem like a benefit to add a folder, with just this file in it.